src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { string, shape, object, arrayOf } from 'prop-types';
+import { Grid, GridItemConstrainedMedium } from '#lib/styledGrid';
+
+const CpsAssetPageMain = () => {
+  return (
+    <Grid as="main" role="main">
+      <GridItemConstrainedMedium>
+        <h1> Placeholder content for MAP page skeleton</h1>
+      </GridItemConstrainedMedium>
+    </Grid>
+  );
+};
+
+CpsAssetPageMain.propTypes = {
+  /* eslint-disable react/no-unused-prop-types */
```

Comment:
question, why are we disabling prop-type warnings here ?

Reply:
got linting error locally `18:13  error  'pageData' PropType is defined but prop is never used  react/no-unused-prop-types`, better to disable this rule for now `  /* eslint-disable react/no-unused-prop-types */ `